### DISCUSSION TITLE
Enlist Frictionless Data as a UN Digital Public Good

### DISCUSSION THREAD

**Discussion by augusto-herrmann** [frictionlessdata/frictionlessdata.io#700] [Ideas]
Digital Public Goods is defined by [unicef/publicgoods-candidates](https://github.com/unicef/publicgoods-candidates/blob/main/README.md) as *open source software, open data, open AI models, open standards and open content that adhere to privacy and other applicable best practices, do no harm and are of high relevance for attainment of [the UN’s 2030 Sustainable Development Goals (SDGs)](https://sdgs.un.org/goals).*

Does that definition apply to Frictionless Data as well?

Could Frictionless Data qualify to become a Digital Public Good?

If so, the page "[Contributing to Digital Public Goods](https://github.com/unicef/publicgoods-candidates/blob/main/CONTRIBUTING.md)" describes how to nominate a codebase for inclusion in Digital Public Goods.

For reference, I think we can use the openfisca/openfisca-core#1056 submission (where I have shamelessly copied the first paragraphs from, thanks @ericherman btw) as a template to find out how to properly fill the submission. Frictionless has in common with OpenFisca the fact that it is also infrastructure, although it is not only a set of software tools, but also standards for data. Digital Public Goods have categories for software, standards and data that I believe are all relevant for Frictionless.

**Comment by lwinfree** [comment]
Thanks @augusto-herrmann I think yes! I'll put this on our to-do list :-)

**Comment by sapetti9** [comment]
We did the submission (thanks @augusto-herrmann !), and we are now waiting for a review:

https://github.com/unicef/publicgoods-candidates/pull/854
https://github.com/unicef/publicgoods-candidates/pull/855

Let's see what happens now!
cc @lwinfree

**Reply by augusto-herrmann** [reply]
Awesome work, @sapetti9 and @lwinfree. Congratulations! :tada: I hope it gets reviewed and accepted soon.
